tests/foreman/installer/test_installer.py

```diff
@@ -1315,6 +1315,31 @@
     '-v',
 }
 
+LAST_SAVED_SECTIONS = {
+    '= Generic:',
+    '= Module certs:',
+    '= Module foreman:',
+    '= Module foreman_cli:',
+    '= Module foreman_compute_ec2:',
+    '= Module foreman_compute_gce:',
+    '= Module foreman_compute_libvirt:',
+    '= Module foreman_compute_openstack:',
+    '= Module foreman_compute_ovirt:',
+    '= Module foreman_compute_vmware:',
+    '= Module foreman_plugin_tasks:',
+    '= Module foreman_proxy:',
+    '= Module foreman_proxy_content:',
+    '= Module foreman_proxy_plugin_ansible:',
+    '= Module foreman_proxy_plugin_dhcp_infoblox:',
+    '= Module foreman_proxy_plugin_dhcp_remote_isc:',
+    '= Module foreman_proxy_plugin_discovery:',
+    '= Module foreman_proxy_plugin_dns_infoblox:',
+    '= Module foreman_proxy_plugin_openscap:',
+    '= Module foreman_proxy_plugin_pulp:',
+    '= Module foreman_proxy_plugin_remote_execution_ssh:',
+    '= Module katello:',
+    '= Module puppet:',
+}
```

Comment:
Subsections verification is missing, Please add subsection verification along with sections.

Reply:
Since we are verifying each and every option separately, I didnt emphasize on subsections. Do you still fill that we should still verify subsections. The gain is very less though.